src/outmdsd/DjsonLogItem.cc

```diff
@@ -64,12 +66,12 @@ DjsonLogItem::ComposeSchema(
         strm << cachedInfo.first << "," << cachedInfo.second;
```

Comment:
>strm << cachedInfo.first << "," << cachedInfo.second; [](start = 8, length = 53)

And you should update the cache to recognize this new (unsorted) order, no?

GetIdMgr.Insert(unsortedKey, std::make_pair(cachedInfo.first, unsortedSchemaArray));